client/homebrew/pages/sharePage/sharePage.jsx

```diff
@@ -103,8 +88,13 @@ const SharePage = createClass({
 				</Nav.section>
 
 				<Nav.section>
-					{this.renderPrint()}
-					{this.renderSourceDropdown()}
+					{this.props.shareId && <PrintLink shareId={this.processShareId()} />
```

Comment:
You can put this on two lines for legibility:

```
{this.props.shareId &&
  <PrintLink shareId={this.processShareId()} />
```

Reply:
Ah, and since React expects only one "element" at a time when we do this, we have to wrap it in an empty "fragment" `<>` so it can process this correctly:

```
<>
  <PrintLink shareId={this.processShareId()} />
  <Nav.item icon='fas fa-code' color='red' className='source'
	onMouseEnter={()=>this.handleDropdown(true)}
	onMouseLeave={()=>this.handleDropdown(false)}>
	source
	{this.renderDropdown()}
  </Nav.item>
</>
```